twitter.html

```diff
@@ -0,0 +1,275 @@
+                        <h4 class="sia">Sia</h4> <span>&#x40;Siasoon · 3d</span>
+                        <p>I just love being a part of this Twitter thing</p>
+                    </blockquote>
+                    <div class="tweet-icons">
+                        <ul>
+                            <li><a href="#comment"><span class="material-symbols-outlined">
+                                chat_bubble
+                                </span></a>
+                            </li>
+    
+                            <li>
+                                <a href="#reply"><span class="material-symbols-outlined">
+                                    reply
+                                    </span></a>
+                            </li>
+    
+                            <li>
+                                <a href="#heart"><span class="material-symbols-outlined">
+                                    favorite
+                                    </span></a>
+                            </li>
+    
+                            <li>
+                                <a href="#share"><span class="material-symbols-outlined">
+                                    ios_share
+                                    </span></a>
+                            </li>
+                        </ul>
+                      </div>
+                </div>
+            </div>
+
+            <!-- right-bar -->
+
+            <div class="right-bar">
+                <label for="search"></label>
```

Comment:
I don't know what this for="" does but it looks interesting / useful, I'll have to look into that!